tests/baselines/reference/for-of39.errors.txt

```diff
@@ -0,0 +1,13 @@
+tests/cases/conformance/es6/for-ofStatements/for-of39.ts(1,15): error TS2453: The type argument for type parameter 'V' cannot be inferred from the usage. Consider specifying the type arguments explicitly.
+  Type argument candidate 'boolean' is not a valid type argument because it is not a supertype of candidate 'number'.
+
+
+==== tests/cases/conformance/es6/for-ofStatements/for-of39.ts (1 errors) ====
+    var map = new Map([["", true], ["", 0]]);

```

Comment:
Perhaps this test would be more interesting if you had given explicit type parameters.